diff -Naru hplip-3.16.7-orig/Makefile.am hplip-3.16.7/Makefile.am
--- hplip-3.16.7-orig/Makefile.am	2016-07-15 10:12:48.000000000 +0000
+++ hplip-3.16.7/Makefile.am	2016-08-16 17:31:24.632699652 +0000
@@ -98,7 +98,7 @@
 libhpmud_la_LDFLAGS += -lusb
 else
 libhpmud_la_SOURCES += io/hpmud/musb.c 
-libhpmud_la_CFLAGS  += -I/usr/include/libusb-1.0
+libhpmud_la_CFLAGS  += -I$(includedir)/libusb-1.0
 libhpmud_la_LDFLAGS += -lusb-1.0
 endif
 
@@ -326,7 +326,7 @@
 endif
 
 if !LIBUSB01_BUILD
-hpmudext_la_CFLAGS +=-I/usr/include/libusb-1.0 
+hpmudext_la_CFLAGS +=-I$(includedir)/libusb-1.0 
 endif
 
 # ui (qt3)
